src/CommunityToolkit.Maui/PlatformConfiguration/AndroidSpecific/NavigationBar.android.cs

```diff
@@ -93,7 +93,7 @@ public static void MapNavigationColorProperty(IPageHandler handler, IContentView
 
 		var color = GetColor(page).ToPlatform();
 
-		if (OperatingSystem.IsAndroidVersionAtLeast(21))
+		if (OperatingSystem.IsAndroidVersionAtLeast(23))
```

Comment:
The minimum supported Android version is API 23 for `NavigationBar`, as noted in the associated `else` block